google-cloud-pubsub/lib/google/cloud/pubsub/subscription/push_config.rb

```diff
@@ -0,0 +1,193 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google/pubsub/v1/pubsub_pb"
+
+module Google
+  module Cloud
+    module PubSub
+      class Subscription
+        # Configuration for a push delivery endpoint.
+        class PushConfig
+          # @private
+          def initialize
+            @grpc = Google::Cloud::PubSub::V1::PushConfig.new
+          end
+
+          # A URL locating the endpoint to which messages should be pushed.
```

Comment:
I would add: "For example, a Webhook endpoint might use "https://example.com/push"." from [here](https://cloud.google.com/pubsub/docs/reference/rpc/google.pubsub.v1#pushconfig).

Reply:
Added.